samples/drivers/i2s/i2s_codec/Kconfig

```diff
@@ -0,0 +1,31 @@
+# Copyright (c) 2024 NXP
+# SPDX-License-Identifier: Apache-2.0
+
+source "Kconfig.zephyr"
+
+config I2S_INIT_BUFFERS
+	int "Initial count of audio data blocks"
+	default 2
+	help
+	  Controls the initial count of audio data blocks, which are (optionally)
+	  filled by data from the DMIC peripheral and played back by the I2S
+	  output perihperal.
+
+config SAMPLE_FREQ
```

Comment:
namespace as I2S_SAMPLE_FREQ?

Reply:
I wouldn't do that, is this frequency is used also for the DMIC peripheral.